### PR TITLE
Get started with 'what is vivarium good for?' page

### PR DESCRIPTION
**Pull request by pletale**
Jira ticket: https://jira.ihme.washington.edu/browse/SSCI-1165 

- sets up different section titles on page
- fills out first section, 'what is individual-based microsimulation?' 

There's obviously still more to work on with this ticket, but thought it would be easier to make smaller PRs! Happy to hear feedback on section titles, order, etc., and wording in first section.

### CODE REVIEW

File: docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst [hunk 1]
```diff
@@ -24,20 +24,54 @@
...
 =========================================================
 
 .. contents::
    :local:
    :depth: 1
 
+What are individual-based microsimulations?
+-------------------------------------------
+
+Vivarium is the Simulation Science team's platform for individual-based microsimulation (also known as agent-based simulation), which is a technique for leveraging information about attributes of individuals in a population (such as demographics, risk factors, disease occurrence, and intervention coverage) and relationships between these attributes to estimate health outcomes in a synthetic environment over time.
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst:37]:
```suggestion
Vivarium, a platform developed by the Simulation Science team, enables individual-based microsimulation, also known as agent-based simulation. This is a technique for combining information about attributes of individuals in a population (such as demographics, risk factors, disease occurrence, and intervention coverage) and relationships between these attributes, to estimate health outcomes in a simulated environment over time.
```

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst (hunk 1 above)
Comment (aflaxman) [reply docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst:37]: I meant to say the "the IHME Simulation Science team"

Comment (zmbc) [inline docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst:37]: nit: it's generally nice to wrap long lines like this into multiple. Makes it a little easier to deal with in git: if someone edits it later, their diff will only be on the lines they touched.

File: docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst [hunk 2]
```diff
@@ -24,20 +24,54 @@
...
+
+- Basic demographics (e.g., age, sex/gender, race/ethnicity)
+- Exposure to risk factors
+- Disease incidence 
+- Healthcare visit frequency
+- Treatment (i.e., timing, duration, frequency, adherence, and known/assumed treatment effects)
+- Adverse events
+- Mortality rate/life expectancy 
+
+By using individual-level microsimulation, we probabilitistically assign the above attributes heterogeneously across individuals within the simulated population. Taken in aggregate, these attributes match real-world population-level data, derived from IHME's Global Burden of Disease (GBD) databases and other sources. Over the duration of a given microsimulation
```
Comment (zmbc) [inline docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst:49]: typo: probabilistically

File: docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst (hunk 1 above)
Comment (zmbc) [reply docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst:37]: Also, maybe for a future PR, but this section uses a lot of big words; it'd be great to simplify and/or add pictures to explain this concept to a microsim newcomer. I think we have PowerPoint slides about this, which might be able to be repurposed here.

**Review by zmbc — APPROVED** [review]
Nice!

**Review by SylLutze — APPROVED** [review]

**Pull request merged**